fields/pandarepeaterfield.php

```diff
@@ -0,0 +1,254 @@
+	?>
+	//call the resize function in the parent iframe, nested iframe only
+	//parent.pprf_resize_fn();
+	
+	<?php
+	}
+	?>
+	pprf_loadedResized_bln = false;
+}
+   
+jQuery(document).ready( function($) {
+	
+	// append a div for click
+	//$( '#wpcontent' ).prepend('<div style="height:100%; width:100%; background:#ccc; position: absolute; top: 0; left: 0; " id="expand-div"></div>');
+/*	$('.toplevel_page_panda-pods-repeater-field').on('click', function(){
+		//console.log('ddd');
+		pprf_resize_fn();
+	});*/
+	
+	//remove update messages
+	$('.updated, .update-nag').remove();
+	// remove admin outlook
+	$('#adminmenuback, #adminmenuwrap, #wpadminbar, #wpfooter, #screen-meta-links, #screen-meta').remove();
+	pprf_resize_fn();
+	$('.pods-form-ui-field-type-pick').on('change', function(){				
+		pprf_resize_fn( );		
+	})
+	//$( this ).on( 'click, mouseenter, mouseout, mouseleave', function(){
+	$( '.click-to-expand, .click-to-expand-arrow' ).on( 'click', function(){
+		
+		// if the iframe has not been resized		
+		//if( pprf_loadedResized_bln == false ){
+			pprf_resize_fn();
+			
+			pprf_loadedResized_bln = true;
+		//}
+	})
+	$( '.click-to-close, .click-to-close-arrow' ).on( 'click', function(){				
+		// if the iframe has not been resized		
+		//if( pprf_loadedResized_bln == true ){
+			pprf_resize_fn( 150 );
+			pprf_loadedResized_bln = false;
+		//}
+	})
+/*	.children().click(function(e) {
+		console.log( e );
+	  //return false;
+	});*/
+	/*$('.wp-toolbar').on( 'click', function(){	
+		 return false;
+	}).children().click(function(e) {
+	  return false;
+	});*/
+	//pprf_resize_fn();
+	//var iframeWin = window.parent.document.getElementById('<?= $iframeID_int ?>-<?= $_GET['poditemid']; ?>').contentWindow;
+	//console.log( iframeWin );
+    //parent.window.addEventListener('resize', function(){
+       // console.log( jQuery('html').height() );
+		//pprf_resize_fn() ;
+    //});	
+	
+	<?php
+	// if successfully added a new one
+	if( isset( $_GET['success'] ) && $_GET['success'] == 1 && isset( $_GET['iframe_id'] ) && strpos( $_GET['iframe_id'], 'panda-repeater-add-new' ) === 0 ){
+		global $wpdb;
+		$lastid = $wpdb->insert_id;
+	?>
+	parent.pprf_new_fn( <?php echo $_GET['podid'];?>, "<?php echo $_GET['postid'];?>", <?php echo $_GET['tb'];?>, <?php echo $current_user->ID;?>, '<?php echo $_GET['iframe_id']; ?>', <?php echo $_GET['poditemid']; ?>, '<?php echo esc_attr( get_the_title( $_GET['poditemid'] ) ); ?>' );
+	<?php	
+	}
+	// disable it until I am sure it is not needed
+	if( 1 == 0 && isset( $_GET['itemid'] ) && is_numeric( $_GET['itemid'] ) ){	
+	?>
+	// tell the parent window to delete this item
+	$('.panda-repeater-field-delete-bn').on('click', function(){
+		parent.pprf_delete_item_fn( <?php echo $_GET['podid'];?>, "<?php echo $_GET['postid'];?>", <?php echo $_GET['tb'];?>, <?php echo $_GET['itemid'];?>, <?php echo $current_user->ID;?>, '<?php echo $_GET['iframe_id']; ?>', <?php echo $_GET['poditemid']; ?> );
```

Comment:
Instead of doing it one by one, I do this:
	if( isset( $_GET ) && count( $_GET ) > 0 ){
		foreach( $_GET as $k_str => $v_ukn ){
			$_GET[ $k_str ]	= esc_attr( $v_ukn );
		}
	}